pages/_app.tsx

```diff
@@ -39,7 +40,11 @@ function collectHeadings(node, sections = []) {
   return sections;
 }
 
-export default function MyApp({ Component, pageProps }: AppProps) {
+export interface MyAppProps {
+  markdoc?: MarkdocNextJsPageProps['markdoc']
+}
+
+export default function MyApp({ Component, pageProps }: AppProps<MyAppProps>) {
```

Comment:
Can we just change this to 
```suggestion
export interface MyAppProps {
  markdoc?: MarkdocNextJsPageProps['markdoc']
}

export default function MyApp({ Component, pageProps }: AppProps<MarkdocNextJsPageProps>) {
```
or
```suggestion
export type MyAppProps = MarkdocNextJsPageProps

export default function MyApp({ Component, pageProps }: AppProps<MyAppProps>) {
```